### PR TITLE
DST - Add 2024 milestones

### PR DESCRIPTION
**Pull request by Zorlin**
This PR adds milestones for 2024 for the Distributed Systems Testing team.

### CODE REVIEW

File: content/vac/dst/deployment-and-analysis/codex/testnet.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      'secondaryColor': '#006100', 
+      'tertiaryColor': '#fff' 
+    } 
+  } 
+}%%
+gantt
+  tickInterval 1month
+  dateFormat YYYY-MM-DD 
+  section Status
+    Testnet: 2024-05-01, 2024-11-01
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/codex/testnet.md]:
```suggestion
    Testnet: 2024-05-01, 2024-12-31
```

We can use the full reporting period for this. The task is quite open ended; we might want a follow up in the next reporting period.

File: content/vac/dst/deployment-and-analysis/codex/testnet.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+- status: 15%
+- CC: Wings
+
+### Description
+
+Assist the Codex team with deploying and running a testnet for the Codex network.
+
+- Provide a 1PiB Storage Provider deployment
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/codex/testnet.md]: Let's say 256 TB first, moving towards 1PiB.

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      'secondaryColor': '#006100', 
+      'tertiaryColor': '#fff' 
+    } 
+  } 
+}%%
+gantt
+  tickInterval 1month
+  dateFormat YYYY-MM-DD 
+  section Status
+    Mixnet: 2024-05-01, 2025-02-01
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/nomos/mixnet.md]:
```suggestion
    Mixnet: 2024-05-01, 2025-02-01
```

The date should be within 2024.
We can define a follow up, if we need. Let's scope this in a way we can fit it in 2024

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+- status: 15%
+- CC: Wings
+
+### Description
+
+Assist the Nomos team with deploying and running a mixnet.
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/nomos/mixnet.md]: I'd say deploying and running at scale in VacLab. (Not in the wild)
We should also do analysis on this.

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+- Provide visualisation tools and work with the Nomos team to implement privacy preserving metrics and measurements in Nomos to help understand the mixnet's performance.
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/nomos/mixnet.md]: Isn't this part of the vis milestone?
You could link to the vis milestone and just mention there will be some integration / interaction here.

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+### Justification
+
+### Deliverables
+- Mixnet fully operational and rolled out
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/nomos/mixnet.md]: This sounds like we roll out the mixnet. But DST will roll out the test in VacLab. (see above)

File: content/vac/dst/deployment-and-analysis/waku/10k.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+- CC: Wings
+
+### Description
+
+Run 10,000 Waku nodes actively passing messages in one network.
+
+Gather bandwidth details and other metrics.
+
+Gather QoS details such as latency, dropped packets, etc.
+
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/waku/10k.md:39]: What about stability?

File: content/vac/dst/deployment-and-analysis/waku/10k.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+- status: 90%
+- CC: Wings
+
+### Description
+
+Run 10,000 Waku nodes actively passing messages in one network.
+
+Gather bandwidth details and other metrics.
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/waku/10k.md]: Other metrics should be defined.

File: content/vac/dst/deployment-and-analysis/waku/10k.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
+---
+title: "10k"
```
Comment (kaiserd) [inline content/vac/dst/deployment-and-analysis/waku/10k.md]: The titles can be more descriptive.
Also, the description tag should be used, too.
(See other milestones)
You can also check Milestones from other Vac units.

**Review by kaiserd — COMMENTED** [review]
Thank you very much. Comments see in-line

File: content/vac/dst/deployment-and-analysis/waku/10k.md (hunk 9 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/waku/10k.md]: I couldn't find the description tag, other than the literal `### Description fields`.

Let's discuss the new draft

File: content/vac/dst/deployment-and-analysis/waku/10k.md (hunk 8 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/waku/10k.md]: Done in new draft

File: content/vac/dst/deployment-and-analysis/waku/10k.md (hunk 7 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/waku/10k.md:39]: Added

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md (hunk 6 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/nomos/mixnet.md]: Clarified it

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md (hunk 5 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/nomos/mixnet.md]: Yep, clarified it, thanks

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md (hunk 4 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/nomos/mixnet.md]: Yep!

File: content/vac/dst/deployment-and-analysis/nomos/mixnet.md (hunk 3 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/nomos/mixnet.md]: Fixed, set it to EOY 2024

File: content/vac/dst/deployment-and-analysis/codex/testnet.md (hunk 2 above)
Comment (Zorlin) [reply content/vac/dst/deployment-and-analysis/codex/testnet.md]: Corrected

File: content/vac/dst/deployment-and-analysis/waku/10k.md (hunk 9 above)
Comment (kaiserd) [reply content/vac/dst/deployment-and-analysis/waku/10k.md]: You can use

```
title: "10k"
description: "10k description test test"
```

**Review by kaiserd — APPROVED** [review]

**Pull request merged**